lib/manageiq/api/common/request.rb

```diff
@@ -8,9 +8,9 @@ def initialize
       end
 
       class Request
-        REQUEST_ID_KEY = "x-request-id".freeze
+        X_RH_INSIGHTS_REQUEST_ID_KEY = "x-rh-insights-request-id".freeze
```

Comment:
can we just keep the constant named REQUST_ID_KEY...it's more readable, and there's no need to make a constant exactly equal to it's value.

Reply:
also the method name is still request_id, so it keeps that parity.